src/modules/user/user/UserAvatar.tsx

```diff
@@ -0,0 +1,26 @@
+import { User } from "@phosphor-icons/react";
+import { Avatar, theme } from "antd";
+
+type UserAvatarProps = {
+  image?: string;
+};
+
+const UserAvatar: React.FC<UserAvatarProps> = ({ image }) => {
+  const {
+    token: { colorPrimary, colorPrimaryBg },
+  } = theme.useToken();
+
+  if (image) {
+    return <Avatar size={44} src={image} alt="User avatar" />;
+  } else {
+    return (
+      <Avatar
+        size={44}
+        style={{ backgroundColor: colorPrimaryBg }}
+        icon={<User color={colorPrimary} weight="duotone" />}
+      />
+    );
+  }
+};
+
+export default UserAvatar;
```

Comment:
fix logic 
avatar to letter type Avatar,color base on user id
uidInt = parseInt(userId, 16)

colors = [ pick some color from this
https://tailwindcss.com/docs/customizing-colors]

usercolor = colors[uidInt % lenColors]